src/inc/Util.class.php

```diff
@@ -782,6 +782,11 @@ public static function sectotime($seconds) {
       $days = floor($seconds / 86400);
       $return = $days . "d ";
       $seconds = $seconds % 86400;
+      if($days > 365.25) { // taken leap year into consideration
+      	$years = floor($days / 365.25);
+      	$days = $days  % 365.25;
+      	$return = number_format($years) . "y " . $days . "d ";
```

Comment:
Would this not yield the same results?
```
if ($seconds >= 86400*365.25) {
    return .= gmdate("Y", $seconds)."y ";
}
if ($seconds >= 31557600) {
    return .= gmdate("d", $seconds)."d ";
}
return $return.gmdate("H:i:s", $seconds);
```

Reply:
There are typos. In both lines within the `if` clause, the `return` should be `$return` instead. No offence, but your suggested code works exactly the same as my code, isn't it?

Reply:
A lot less math is involved and it's in line with other code, using the same functions